src/components/ui.js

```diff
@@ -355,6 +356,12 @@ AFRAME.registerComponent('ui', {
     this.handEl.setAttribute('brush', 'size', brushSize);
   },
 
+  toggleCaptureCamera: function() {
+    var enableCamera = this.handEl.getAttribute('trigger-mode') != 'camera'
+    this.handEl.setAttribute('trigger-mode', enableCamera ? 'camera' : 'brush')
+    this.handEl.emit('trigger-mode-changed')
```

Comment:
Is there a built-in way to listen for any attribute changes, or do i need to emit custom events each time i change?